Bio/SCOP/__init__.py

```diff
@@ -573,8 +571,8 @@ def getParent(self):
             return self.scop.getNodeBySunid(self.parent)
 
     def getDescendents(self, node_type):
-        """ Return a list of all descendant nodes of the given type. Node type can a
-        two letter code or longer description. e.g. 'fa' or 'family'
+        """Return a list of all descendant nodes of the given type. Node type can be a
```

Comment:
Since you'd editing this docstring & will have to come back to this file, I'd insert the line break and one blank line after the first sentence:

```python
"""Return a list of all descendant nodes of the given type.

Node type can be a two letter code or longer description.
e.g. 'fa' or 'family'.
"""
```